Extensions/GooglePlayGameServices/JsExtension.js

```diff
@@ -0,0 +1,142 @@
+      .setIcon('https://developers.google.com/static/games/services/images/not-yet-friends.svg');
+
+    extension
+      .registerProperty('ApplicationId')
+      .setLabel(_('Application ID'))
+      .setDescription('The application ID string consists only of the digits (typically 12 or more) at the beginning of the client ID provided by the Google Play Console')
+      .setType('string');
+
+    extension
+      .addDependency()
+      .setName('Cordova Android Play Services Gradle')
+      .setDependencyType('cordova')
+      .setExportName('cordova-android-play-services-gradle-release')
+      .setVersion('4.0.0');
+
+    extension
+      .addDependency()
+      .setName('Cordova Play Games Services Plugin')
+      .setDependencyType('cordova')
+      .setExportName('cordova-plugin-play-games-services')
+      .setVersion('1.1.2')
+      .setExtraSetting(
+        'APP_ID',
+        new gd.PropertyDescriptor('ApplicationId').setType(
+          'ExtensionProperty'
+        )
+      )
+      .onlyIfOtherDependencyIsExported('Cordova Android Play Services Gradle');
+
+    // Banner
+    extension
+      .addCondition(
+        'GameServicesReadyAndroid',
+        _('Google Play Games Services Android is ready'),
+        _(
+          'Checks if google play games services Android is ready.'
+        ),
+        _('Google Play Games Services Android is ready'),
+        '',
+        'https://developers.google.com/static/games/services/images/not-yet-friends.svg',
+        'https://developers.google.com/static/games/services/images/not-yet-friends.svg'
+      )
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/googlePlayGameServices/tools.js')
+      .setFunctionName('gdjs.evtTools.googlePlayGameServices.gameServicesReady');
+
+    extension
+      .addAction(
+        'GooglePlayServicesAndroidAPI',
+        _('Call Google Play Services Android API'),
+        _('Call Google Play Services Android API.'),
+        _('Call Google Play Services Android API method _PARAM0_ then save results to scene variable _PARAM1_'),
+        '',
+        'https://developers.google.com/static/games/services/images/not-yet-friends.svg',
+        'https://developers.google.com/static/games/services/images/not-yet-friends.svg'
+      )
+      .addParameter('string', _('Method'), '', false)
+      .setParameterLongDescription(
+        'Method'
+      )
+      .addParameter('string', _('ResultSceneVariable'), '', false)
+      .setParameterLongDescription(
+        'ResultSceneVariable'
+      )
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/googlePlayGameServices/tools.js')
+      .setFunctionName('gdjs.evtTools.googlePlayGameServices.gameServicesReady');
+
+    extension
+      .addAction(
+        'GooglePlayServicesAndroidAPIwParameters',
+        _('Call Google Play Services Android API w/ Parameters'),
+        _('Call Google Play Services Android API w/ Parameters.'),
```

Comment:
It's the same here. I think you should replace w/ by with for more clarity. I'm not 100% certain abbrevations are allowed or not...